mistletoe/latex_renderer.py

```diff
@@ -57,15 +57,21 @@ def render_image(self, token):
         self.packages['graphicx'] = []
         return '\n\\includegraphics{{{}}}\n'.format(token.src)
 
+    @staticmethod
+    def render_url(token):
+        return token.target.replace('%', '\\%') \
+                           .replace('#', '\\#')
+
     def render_link(self, token):
         self.packages['hyperref'] = []
         template = '\\href{{{target}}}{{{inner}}}'
         inner = self.render_inner(token)
-        return template.format(target=token.target, inner=inner)
+        return template.format(target=self.render_url(token),
```

Comment:
OK, but what about escaping the `inner` part as well?

Reply:
I just double-checked that the `inner` part is indeed properly escaped as is. `self.render_inner` eventually calls `self.render_raw_text` which takes care of that .

Reply:
You're right, false alarm here.